hres_taught_schema/requirements.schema

```diff
@@ -0,0 +1,115 @@
+
+# -------- Attributes ------------------------------
+
+attribute hres:attribute:researcher as Researcher
+attribute hres:attribute:supervisor as Supervisor
+attribute hres:attribute:academic-year as AcademicYear
+attribute hres:attribute:research-institute as ResearchInstitute
+
+attribute hres:attribute:taught-student as TaughtStudent
+    title: Taught Student
+    search-name taught-student
+    qualifier std:qualifier:null
+    data-type link
+    linked-type hres:type:person:taught-student
+
+attribute hres:attribute:course-module as CourseModule
+    title: Course Module
+    search-name: course-module
+    qualifier std:qualifier:null
+    data-type link
+    linked-type hres:type:course-module
+
+attribute hres:attribute:module-leader as ModuleLeader
+    title: Module Leader
+    search-name: module-leader
+    qualifier std:qualifier:null
+    data-type link
+    linked-type hres:type:person:researcher
+
+attribute hres:attribute:code as Code
+    title: Code
+    search-name: code
+    qualifier std:qualifier:null
+    data-type text
+
+# -------- Types -----------------------------------
+
+type hres:type:person:taught-student as TaughtStudent
+    title: Taught Student
+    parent-type std:type:person
```

Comment:
Currently HRES defines Researcher, Doc Researcher, Staff etc. all as subtypes of the standard person type - made more sense for Taught Students to also subtype this instead of being it's own root. Conceptually all people?